src/Arcus.BackgroundJobs.DataBricks/DatabricksInfoProvider.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Threading.Tasks;
+using GuardNet;
+using Microsoft.Azure.Databricks.Client;
+
+namespace Arcus.BackgroundJobs.DataBricks
+{
+    /// <summary>
+    /// Provides dev-friendly access to the Databricks instance.
+    /// </summary>
+    internal class DatabricksInfoProvider : IDisposable
```

Comment:
Why not make this public?

Reply:
Hmm, bc it was only used here, in the background job. 🤔 .

Reply:
That's correct but we shouldn't block people who want to use it somewhere else, for example in a Function